### PR TITLE
Pass props down to Modal.

### PR DESCRIPTION
**Pull request by rcorral**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @kennyt

File: src/Confirm/Confirm.js [hunk 1]
```diff
@@ -21,17 +22,21 @@ export default class Confirm extends React.Component {
   }
 
   render() {
+    let props = Util.exclude(this.props, 'children', 'disabled',
+      'leftButtonText', 'leftButtonClassName', 'leftButtonCallback',
+      'rightButtonText', 'rightButtonClassName', 'rightButtonCallback'
+      );
```
Comment (kennyt) [inline src/Confirm/Confirm.js]: indented a little too far

**Comment by kennyt** [comment]
konflicts

File: src/Confirm/Confirm.js (hunk 1 above)
Comment (rcorral) [inline src/Confirm/Confirm.js]: fixed

**Comment by kennyt** [comment]
:ship:

**Pull request merged**